ext/pdo/php_pdo.h

```diff
@@ -49,17 +49,18 @@ extern zend_module_entry pdo_module_entry;
 
 PHP_MINIT_FUNCTION(pdo);
 PHP_MSHUTDOWN_FUNCTION(pdo);
+PHP_RINIT_FUNCTION(pdo);
+PHP_RSHUTDOWN_FUNCTION(pdo);
 PHP_MINFO_FUNCTION(pdo);
 
 ZEND_BEGIN_MODULE_GLOBALS(pdo)
 	zend_long  global_value;
+	HashTable  *pdo_pdbh_to_delete;
```

Comment:
@weltling If that's the case, can we please have an announcement on internals? I assumed that ABI breaks (not necessarily source breaks) are fine until the final release, similar to how it has been with previous versions.